test/web-platform-tests/to-upstream/dom/nodes/ParentNode-querySelector-escapes.html

```diff
@@ -65,7 +65,6 @@
 testMatched("outOfRange\u{fffd}", "#outOfRange\\ffffff");
 
 // - escape EOF
-testMatched("eof\u{fffd}", "#eof\\");
```

Comment:
Why remove this?

Reply:
The same test already exists above, leading to the same test name passed to `test()` in both instances. testharness now rejects tests with the same name.